apps/webapp/src/components/send/ibc/ibc-form.tsx

```diff
@@ -72,21 +72,21 @@ export default function IbcForm() {
           },
         ]}
         balances={accountBalances}
-        /**
-         * @todo: Pass a real fee here once we're actually using this form. See
-         * `<SendForm />` for an example of how to do this.
-         */
-        fee={undefined}
       />
       <ChainSelector />
       <InputBlock
         label='Recipient on destination chain'
-        placeholder='Enter the address'
         className='mb-1'
         value={destinationChainAddress}
-        onChange={e => setDestinationChainAddress(e.target.value)}
         validations={[penumbraAddrValidation()]}
-      />
+      >
+        <Input
+          variant='transparent'
+          placeholder='Enter the address'
+          value={destinationChainAddress}
+          onChange={e => setDestinationChainAddress(e.target.value)}
+        />
+      </InputBlock>
```

Comment:
`InputBlock` no longer renders its own `Input`, so we have to render it ourselves as a child.